configs/default.json

```diff
@@ -20,16 +20,16 @@
       { "format": "qam", "order": 16, "label": "16QAM" },
       { "format": "qam", "order": 64, "label": "64QAM" }
     ],
-    "max_signals": 10,
+    "max_signals": 100,
```

Comment:
Is this change intentional? Looking at the outputs generated by this config file, only ~20 signals are actually generated in each burst list due to collisions.